database/migrations/20190221115302_changeforeignkey.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.table('contacts', table => {
+    table.renameColumn('contacts_id', 'id');
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropColumn('id');
```

Comment:
You shouldn't be dropping the column. Since you're renaming, you should rename it back to what it was originally.